virt/virtual_machines/advanced_vm_management/virt-enabling-descheduler-evictions.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="virt-enabling-descheduler-evictions"]
+= Enabling descheduler evictions on virtual machines
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-enabling-descheduler-evictions
+
+toc::[]
+
+:FeatureName: Descheduler eviction for virtual machines
+include::snippets/technology-preview.adoc[]
+
+The xref:../../../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler[descheduler] can be used to evict a running pod to allow the pod to be rescheduled onto a more suitable node. You must install the descheduler in the {product-title} web console or OpenShift CLI (`oc`) before you can enable it on your VM.
```

Comment:
s/in the {product-title}/by using the {product-title}

s/VM/virtual machine (VM)